### PR TITLE
Merge sister updates to maap develop

### PR DESCRIPTION
**Pull request by sujen1412**

### CODE REVIEW

**Comment by sujen1412** [comment]
@NamrataM - Could you please verify that this PR has the required changes for bringing in updates from sister to MAAP. 

@bsatoriu - Could you please verify that these new updates do not seem to break any current functionality. 

I will be testing a deployment of this api feature branch on DIT before merging as well.

**Comment by sujen1412** [comment]
Should ideally be merged with https://github.com/MAAP-Project/maap-py/pull/45.

File: api/endpoints/cmr.py [hunk 1]
```diff
@@ -24,6 +25,130 @@
 ns = api.namespace('cmr', description='Operations related to CMR')
 
 
+@ns.route('/granules')
```
Comment (bsatoriu) [inline api/endpoints/cmr.py:28]: This CmrGranules class is a duplicate of the same class on line 330

Comment (sujen1412) [reply api/endpoints/cmr.py]: Reverted the cmr.py to whats on develop.

**Review by bsatoriu — APPROVED** [review]

File: api/utils/hysds_util.py [hunk 2]
```diff
@@ -645,12 +698,16 @@ def revoke_mozart_job(job_id):
...
-    submit_response = mozart_submit_job(job_type=job_type, params=params, queue = settings.LW_QUEUE)
+    submit_response = mozart_submit_job(job_type=job_type, params=params, queue=settings.LW_QUEUE)
     lw_job_id = submit_response.get("result")
     logging.info(lw_job_id)
 
     # keep polling mozart until the purge job is finished.
     return poll_for_completion(lw_job_id)
 
 
+def pele_get_product_by_id(id):
```
[the remark this answers is not in the document]
Comment (sujen1412) [reply api/utils/hysds_util.py:710]: I think this was added as a future support to query products from the DPS database.

**Pull request merged**